metrics/repl.lag/lag.go

```diff
@@ -54,28 +68,29 @@ func (c *Lag) Help() blip.CollectorHelp {
 		Domain:      DOMAIN,
 		Description: "Replication lag",
 		Options: map[string]blip.CollectorHelpOption{
-			OPT_WRITER: {
-				Name:    OPT_WRITER,
-				Desc:    "Type of heartbeat writer",
-				Default: "blip",
+			OPT_LAG_SOURCE: {
+				Name:    OPT_LAG_SOURCE,
+				Desc:    "How to collect Lag from",
+				Default: "auto",
 				Values: map[string]string{
-					"blip": "Native Blip replication lag",
-					//"pfs":    "MySQL Performance Schema",
+					"auto": "Auto-determine best source",
+					"blip": "Native Blip heartbeat replication lag",
+					"pfs":  "MySQL8 Performance Schema",
 					///"legacy": "Second_Behind_Slave|Replica from SHOW SHOW|REPLICA STATUS",
```

Comment:
I am still skipping this as I am not sure if it's going to be useful given the other options. But I can implement it as well for the sake of completeness.